common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -1274,6 +1329,23 @@ public static String normalizedOs() {
         return NORMALIZED_OS;
     }
 
+    public static Set<String> normalizedLinuxClassifiers() {
+        return Collections.unmodifiableSet(LINUX_OS_CLASSIFIERS);
```

Comment:
Why not create this once in the static method and return the same instance all the time ?

Reply:
good idea - I'll do it in the static initializer though